src/app/pages/ParaTimesPage/TransactionConfirmation/index.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useContext } from 'react'
+import { Box, CheckBox, Form, FormField, ResponsiveContext, Text } from 'grommet'
+import { Trans, useTranslation } from 'react-i18next'
+import { ParaTimeContent } from '../ParaTimeContent'
+import { ParaTimeFormFooter } from '../ParaTimeFormFooter'
+import { useParaTimes } from '../useParaTimes'
+import { useParaTimesNavigation } from '../useParaTimesNavigation'
+
+export const TransactionConfirmation = () => {
+  const { t } = useTranslation()
+  const { isDepositing, isEvmcParaTime, paraTimeName, setTransactionForm, transactionForm } = useParaTimes()
+  const { navigateToAmount, navigateToSummary } = useParaTimesNavigation()
+  const isMobile = useContext(ResponsiveContext) === 'small'
+
+  return (
+    <ParaTimeContent
+      description={
+        <Trans
+          i18nKey="paraTimes.confirmation.description"
+          t={t}
+          values={{
+            actionType: isDepositing
+              ? t('paraTimes.confirmation.receiving', 'into the receiving')
+              : t('paraTimes.confirmation.withdrawing', 'from the withdrawing'),
+            address: transactionForm.recipient,
```

Comment:
This is empty when withdrawing from EVM.

Unless you first go through withdrawing from cipher, click "Previous", and withdraw from Emerald: then it displays a cipher address :scream:

Reply:
EMVc withdraws need an extra step with recipient address. will add it to TODO list and fix while working on withdraws.